API/PROBLEMS/things_to_check/metavariable

```diff
@@ -0,0 +1,16 @@
+We should clarify:
+- Why do we need metavariables ?
+- What is their relation (if any) to existential variables ?
+- What is their relation (if any) to Constrexpr.CHole ?
+- Why should Kernel care about them?
+  (Constr,Term)
+- Why they must be part of EConstr?
+
+--------------------------------------------------------------------------------
+
+INSTANCES:
+- Metavariable.*
+- Term.mkMeta
+- Term.subst_meta
+- EConstr.pop
+- EConstr.mk_new_meta
```

Comment:
Metavariables are an antique feature that Matthieu is working on getting rid of. They are currently used in the unification algorithm `w_unify`.